src/common/snippets/include/snippets/generator.hpp

```diff
@@ -23,26 +24,21 @@ namespace snippets {
  */
 class Schedule {
 public:
-    /**
-     * @brief Default constructor
-     */
-    Schedule() : work_size({}), is_flat(false), ptr(nullptr) {}
+    Schedule() = default;
     /**
      * @brief Default to create schedule out of specific parameters
-     * @param ws work size for kernel execution
-     * @param f can this kernel be linearided to 1D range
+     * @param wd work domain for kernel execution
      * @param p pointer to generated code
      */
-    Schedule(const ov::PartialShape& ws, bool f, code p) : work_size(ws), is_flat(f), ptr(p) {}
+    Schedule(const VectorDims& wd, code p) : parallel_exec_domain(wd), ptr(p) {}
     /**
      * @brief Returns callable instanse of code pointer
      */
     template<typename K> K get_callable() const {
         return reinterpret_cast<K>(const_cast<unsigned char*>(ptr));
     }
 
-    ov::PartialShape work_size {};
-    bool is_flat {false};
+    VectorDims parallel_exec_domain {};
```

Comment:
Again about wording:
We have `exec_domain`, `work_domain`, `master_shape`, Is it poosible to unify them?

Reply:
Agree, `exec_domain`, `work_domain`, `parallel_exec_domain` refer to the same entity so I renamed them to `parallel_exec_domain`. 
`master_shape` is a bit different thing, since it represents shapes only and doesn't care about execution limitations. For example, it can have a rank smaller than `loop_depth`. We can transform `master_shape` to `parallel_exec_domain`, but not the other way, so I'm not 100% sure we are ready to remove it. However, we should definitely do it in future (after `loop_depth` will be deprecated).